README.md

```diff
@@ -233,19 +233,19 @@ Below is a short synopsis of the CLI commands. For detailed options, specify the
 
 #### everyauth init
 
-Performs one-time initialization of EveryAuth on a developer machine. This command will create a free Fusebit account and store the credentials necessary to access it in the `~/.fusebit/settings.json` file in your home directory. Keep this file secret. You can also move this file to a new machine where you want to access your EveryAuth configuration from, like a CI/CD box or a second development machine. 
+Performs one-time initialization of EveryAuth on a developer machine. This command will create a free Fusebit account and store the credentials necessary to access it in your home directory's `~/.fusebit/settings.json` file. Keep this file secret. You can also move this file to a new machine from which you want to access your EveryAuth configuration, like a CI/CD box or a second development machine. 
```

Comment:
You can also move this file ... -> You can also move the `.fusebit` directory...

Reply:
Fixed